src/components/Blog/PostLayout.tsx

```diff
@@ -111,8 +101,20 @@ export const PostLayout: FunctionComponent<PostComponentProps> = ({
             </div>
         )}
 
-        <ContentSection>
-            <TrySourcegraph stackedButtons={true} />
-        </ContentSection>
+        <CtaSection
+            centerContent={true}
+            title="Try Sourcegraph for free today"
+            description="You'll be searching your own code in 10 minutes. You can run it self-hosted (all of your code stays local and secure)."
+            cta1={{
+                text: 'Try Sourcegraph now',
+                link: '/get-started/self-hosted',
+                ctaStyle: 'primaryButton',
+            }}
+            cta2={{
+                text: 'Schedule a demo',
+                link: '/demo',
+                ctaStyle: 'outlineButton',
+            }}
+        />
```

Comment:
Flagging that trading out the Blog CTA makes this section a lot longer than it was previously and on the DLS. I think on the blog it makes sense for this section to fit the blog's feel as it's different from the rest of the site. What're your thoughts?

<img width="955" alt="image" src="https://user-images.githubusercontent.com/59381432/187727521-3be4fc9d-e36d-4298-9a4b-c76ba20c5f30.png">

Reply:
Yeah I agree about fitting the blog's "feel." I think we can make this work with a few revisions. Working with Tracey atm to solidify it and will update shortly!

Reply:
@st0nebraker ok, all set now! Check out a blog post for the new slim width CtaSection variant. We also changed the blog content width to align with the DLS so we're all matched up now.